arch/risc-v/src/mpfs/crypto.defs

```diff
@@ -0,0 +1,14 @@
+MPFS_CRYPTO = mpfs/crypto/.git
+$(MPFS_CRYPTO):
+	$(Q) echo "Cloning PolarFire crypto driver"
+	$(Q) git clone git@github.com:tiiuae/pf_crypto mpfs/crypto
```

Comment:
Not sure if cloning via https would be better ? No need to setup SSH keys.

Reply:
That is a private repo, so access needs authentication. I was thinking that like that it works with the same ssh keys as what are used for saluki etc...